index.html

```diff
@@ -7,50 +7,219 @@
     <title>Alex Portfolio</title>
     
     <!--Style-->
-    <link rel="stylesheet" href="./style/style.css"> 
+    <link rel="stylesheet" href="./style/dark-style.css"> 
+    <link rel="stylesheet" href="./style/light-style.css"> 
 
     <!--Roboto-->
     <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
     <!--Crete round-->
     <link href='https://fonts.googleapis.com/css?family=Crete%20Round' rel='stylesheet'>
-    <!--Crete round-->
+    <!--Poppins-->
     <link href='https://fonts.googleapis.com/css2?family=Poppins' rel='stylesheet'>
+    <!--Inter-->
+    <link href='https://fonts.googleapis.com/css2?family=Inter' rel='stylesheet'>
 
     <!-- Add icon library -->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-
 </head>
 <body>
-    <header>
+    <!--Main Mobile-->
+    <div class="dark">
+        <header>
+
+            <nav>
+                <ul>
+                    <li >
+                        <a class="my-logo" href="#">Alex Dev</a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="white" class="bi bi-list" viewBox="0 0 16 16">
+                                <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+                            </svg>
+                        </a>
+                    </li>
+                </ul> 
+            </nav>
+
+        </header>
+        <section>
+            <h1>Hey There. I'm Alex</h1>
+            <h2>I'm a Software Developer</h2>
+            <p>I can help you build a product, feature or website Look trough some of my work and experience! If you like what you see and have a project you need coded, don't heastiate contact me.</p>
+        </section>
+        <footer>
+            <a href="#" class="fa fa-github fa-lg"></a>
+            <a href="#" class="fa fa-linkedin fa-lg"></a>
+            <a href="#" class="fa fa-angellist fa-lg"></a>
+            <a href="#" class="fa fa-twitter fa-lg"></a>
+            <a href="#" class="fa fa-medium fa-lg"></a>
+        </footer>
+    </div>
+    
+    <!--Works-->
+    <div class="light">
+        <header>
+            <h1>My Recent Works</h1>
+            <hr>
+        </header>
+        <section>
```

Comment:
I don't get what do you mean with: 
"I think it's better to use a <section> for the work section instead of a <div>. To make good use of HTML semantic tags."